### PR TITLE
[Realise layout, Backend Service] from develop to master

### PR DESCRIPTION
**Pull request by DarkProDen**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "node-fetch": "^2.6.0",
```
Comment (pushharder) [inline package.json]: don't need that

File: src/components/App/App.js [hunk 2]
```diff
@@ -4,44 +4,52 @@ import NavMenu from '../NavMenu';
...
-        {this.state.planetsListError ? <div>Error :(</div> : <Slideshow />}
+        <NavMenu />
+        <Slideshow />
         <div className="unit-wrap">
           {this.state.unitsListError ? (
-            <div>Error :(</div>
+            <ErrorStar errorMessage={this.state.unitsListErrorMessage} />
           ) : this.state.unitsList.length === 0 ? (
-            <div>Loading...</div>
+            <Loader />
```
Comment (pushharder) [inline src/components/App/App.js:52]: move the logic to a function

File: src/components/Loader/Loader.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './Loader.css';
+
+function Loader(props) {
+  return (
+    <div className="loader">
+      <div className="ldio-3vks3r6mm9z">
+        <div></div>
```
Comment (pushharder) [inline src/components/Loader/Loader.js:8]: why 2 divs?

**Review by pushharder — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (DarkProDen) [reply package.json]: resolved

File: src/components/Loader/Loader.js (hunk 3 above)
Comment (DarkProDen) [reply src/components/Loader/Loader.js:8]: to align to center

**Pull request merged**